environments/ithc/ithc.tfvars

```diff
@@ -356,6 +356,12 @@ frontends = [
     backend_domain = ["firewall-nonprodi-palo-ithc.uksouth.cloudapp.azure.com"]
 
   },
+  {
+    name             = "rpts"
+    mode             = "Detection"
+    custom_domain    = "rpts.ithc.platform.hmcts.net"
```

Comment:
have you added your public dns? https://hmcts.github.io/ways-of-working/path-to-live/public-dns.html#external-dns

Reply:
yes under https://github.com/hmcts/azure-public-dns/pull/775